This patch removes two icons from the documentation which would be loaded from
remote servers, thereby exposing the reader's IP address to potential tracking
(lintian keywords privacy-breach-logo and privacy-breach-w3c-valid-html).

Author: Stefan Ott <stefan@ott.net>
--- a/doc/index.html
+++ b/doc/index.html
@@ -427,24 +427,5 @@
         </td>
       </tr>
     </table>
-    
-    <br><br>
-    
-    <table align="center" width="80%" border="0">
-      <tr>
-        <td align="left">
-          <a href="http://sourceforge.net">
-            <IMG src="http://sourceforge.net/sflogo.php?group_id=979&amp;type=1" width="88" height="31" border="0" alt="SourceForge.net"></A>
-        </td>
-        <td align="right">
-
-          <a href="http://validator.w3.org/check/referer">
-            <img border=0
-              src="http://validator.w3.org/images/vh401"
-              alt="Valid HTML 4.01!" height=31 width=88></a>
-        </td>
-      </tr>
-    </table>
-    <br>
   </body>
 </html>
